MTV 2.6.0: quick grammar fix (#501)
kubev2v/forklift-documentation: documentation/modules/new-migrating-virtual-machines-cli.adoc

+
[NOTE]
====
* If the migrated machines is set with a custom CPU model, it will be set with that CPU model in the destination cluster, regardless of the setting of `preserveClusterCpuModel`.

* If the migrated machine is _not_ set with a custom CPU model:

** If `preserveClusterCpuModel` is set to 'true`, {project-short} checks the CPU model of the VM when it runs in {rhv-short}, based on the cluster's configuration, and then sets the migrated VM with that CPU model.

> ** If `preserveClusterCpuModel` is set to 'false`, {project-short} does not set a CPU type and the VMs are set with the default CPU model of the destination cluster.

Author: ```suggestion
** If `preserveClusterCpuModel` is set to 'false`, {project-short} does not set a CPU type and the VM is set with the default CPU model of the destination cluster.
```

====

> * If the migrated machines are set with a custom CPU model, they will be set with that CPU model in the destination cluster, regardless of the setting of `preserveClusterCpuModel`.

Author: ```suggestion
* If the migrated machines is set with a custom CPU model, it will be set with that CPU model in the destination cluster, regardless of the setting of `preserveClusterCpuModel`.
```
